transport/grpc/handler.go

```diff
@@ -49,6 +50,21 @@ func getTRequest(ctx context.Context, msgBodyDecoder func(interface{}) error) (*
 		return nil, err
 	}
 
+	ctxMetadata, err := getContextMetadata(ctx)
```

Comment:
Nit: consider passing it around as just `md` instead of `ctxMetadata` since
that's the only metadata supported.